docs/earthfile/earthfile.md

```diff
@@ -677,6 +677,14 @@ SAVE IMAGE my-example-registry.com/another-image:latest
 ```
 {% endhint %}
 
+{% hint style='danger' %}
+##### Important
+
+Since the introduction of the `--referenced-save-only` feature flag, images are only saved if they are connected to the initial target through a chain of `BUILD` commands. [See the `--referenced-save-only` flag documentation for more details](./features.md#referenced-save-only).
```

Comment:
I think it would be helpful to call out this is enabled via `VERSION 0.6`

Reply:
That was the point of the link; I was trying to not pull in so many details of how our feature flags work. I gave it a crack in the latest commit

Reply:
I added something but I will use your wording I guess